tf2onnx/graph.py

```diff
@@ -1044,21 +1047,29 @@ class GraphUtil(object):
     """Utilities for Graph manipulation."""
 
     @staticmethod
-    def optimize_graph(graph, doc_string, optimize=None, debug=False):
+    def optimize_graph(graph, debug=False):
         """Optimize the graph, for example: eliminating all useless Transpose/Identity pairs.
 
         Returns:
             model proto after optimization, if optimizer run successfully
             or None, if exceptions happen
         """
         try:
-            opts = [TransposeOptimizer(graph, output_names=graph.outputs, debug=debug),
-                    IdentityOptimizer(graph, output_names=graph.outputs, debug=debug)
+            opts = [TransposeOptimizer,
```

Comment:
i am considering to move details graph optimization logic to tf2onnx/optimizer.
and here we only call things like ```optimzer.optimize_graph```

Reply:
let's register optimizers with {name: optimizer} way?

Reply:
move it to "optimzer.optimize_graph" is ok to me. 
i suggested to do it before.